include/cbl++/Collection.hh

```diff
@@ -202,4 +202,13 @@ namespace cbl {
     };
 }
 
+/** Hash function for Collection. */
+template<> struct std::hash<cbl::Collection> {
```

Comment:
Why is this a template?

Reply:
It's a "custom specialization of std::hash can be injected in namespace std" based on https://en.cppreference.com/w/cpp/utility/hash. I have also seen a couple places in LiteCore doing the same.